scipy/interpolate/ndgriddata.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, x, y, rescale=False, tree_options=None):
         if tree_options is None:
             tree_options = dict()
         self.tree = cKDTree(self.points, **tree_options)
-        self.values = y
+        self.values = np.array(y)
```

Comment:
np.asarray(y)

Reply:
Thank you @pv. 
I think that the better performance.